ops/terraform/services/base/values/test.yaml

```diff
@@ -111,8 +111,12 @@
 /bfd/${env}/common/nonsensitive/rds_master_username: bfduser
 /bfd/${env}/common/nonsensitive/rds_cluster_identifier: bfd-test-aurora-cluster
 /bfd/${env}/common/nonsensitive/rds_instance_class: db.r6i.2xlarge
-/bfd/${env}/common/nonsensitive/rds_instance_count: "1"
 /bfd/${env}/common/nonsensitive/rds_security_group: bfd-test-aurora-cluster
+/bfd/${env}/common/nonsensitive/rds_min_reader_nodes: 0
+/bfd/${env}/common/nonsensitive/rds_max_reader_nodes: 0
```

Comment:
I _think_ the existing scheduled scaling in `test` is _supposed_ to give us a single reader node, but scheduled action has been suspended on the `test` cluster so we've been running with a single writer for some time. Given that fact, I've disabled reader nodes in `test`. Should we instead have a single reader node (we set the min and max to `1`)?

Reply:
I think this is fine. With the pre-purchase agreement going into effect in June, without **this** work... I went ahead and disabled the scale-out/scale-in schedules.